src/pages/CustomLessonsDisplay/CustomLessonsDisplay.js

```diff
@@ -1,67 +1,86 @@
 import React, { useState, useEffect } from "react";
+import { DropdownButton, Dropdown } from "react-bootstrap";
 import { withFirebase } from "utils/Firebase";
-import { TEMPLATE_LESSON_MAP, ADMIN_ACCOUNT } from "utils/constants.js";
+import {
+  TEMPLATE_LESSON_MAP,
+  TEMPLATE_WORD_GROUPS,
+  ADMIN_ACCOUNT
+} from "utils/constants.js";
 import "./CustomLessonsDisplay.scss";
 import { compose } from "recompose";
+import { Col, Row } from "reactstrap";
 import { withRouter, Redirect, useHistory } from "react-router-dom";
 import LessonDateDisplay from "../../components/LessonDateDisplay/LessonDateDisplay";
 import LessonInfoDisplay from "../../components/LessonInfoDisplay/LessonInfoDisplay";
 import LessonNameDisplay from "../../components/LessonNameDisplay/LessonNameDisplay";
 
 const CustomLessonsDisplay = props => {
   const { firebase } = props;
```

Comment:
can you update this to be `function CustomLessonsDisplay({ firebase }) {` and then delete the declaration of firebase

Reply:
hello